wsmaster/che-core-api-devfile/src/main/java/org/eclipse/che/api/devfile/server/convert/tool/kubernetes/KubernetesToolToWorkspaceApplier.java

```diff
@@ -233,4 +200,28 @@ private String asYaml(Tool tool, List<HasMetadata> list) throws DevfileRecipeFor
           e);
     }
   }
+
+  private void applyEntrypoints(List<Entrypoint> entrypoints, List<HasMetadata> list) {
+    entrypoints.forEach(ep -> applyEntrypoint(ep, list));
+  }
+
+  private void applyEntrypoint(Entrypoint entrypoint, List<HasMetadata> list) {
+    ContainerSearch search =
+        new ContainerSearch(
+            entrypoint.getDeployment(),
+            entrypoint.getPod(),
+            entrypoint.getContainer(),
+            entrypoint.getPodSelector(),
+            entrypoint.getDeploymentSelector());
+
+    List<Container> cs = search.search(list);
+
+    List<String> command = entrypoint.getCommand();
```

Comment:
`entrypoint.getCommand()` retuns empty list even if user does not define it.
Should not we override command/arg if it is not specified in Devfile?